src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+
+>***Note 1***
+>When referring to the `uid` of tracker objects, they are labeled as their object names in the payload. For example, the `uid` of a tracked entity would in the payload have the name "trackedEntity". The same goes for "enrollment", "event" and "relationship" for enrollments, events and relationships respectively.
+
+>***Note 2***
+>If no uid is provided in the payload, the import process will generate new uids. This means the error report might refer to a uid that does not exist in your payload.
+
+>***Note 3***
+>Errors are represents issues with the payload which the importer can not circumvent. Any errors will block that data from being imported. Warnings on the other hand are issues where it's safe to circumvent them, but the user should be made aware that it happened. Warnings will not block data from being imported.
+
+***stats***
+
+The status provides a quick overview of the import. After an import is completed, this will be the actual counts representing how much data was created, updated, deleted or ignored.
+
+Example:
+```json
+{
+  "stats": {
+    "created": 2,
+    "updated": 2,
+    "deleted": 1,
+    "ignored": 5,
+    "total": 10
+  }
+}
+```
+`created` refers to how many new objects which was created. In general, objects without an existing uid in the payload will be treated as a new object.
+
+`updated` refers to the number of objects updated. Objects with an existing uid in the payload will be treated as updating an existing object.
+
+`deleted` refers to the amount of objects deleted during the import. Deletion only happens when the import is configured to delete data, and only then when the objects in the payload have existing uids set.
+
+`ignored` refers to objects that was not persisted. Objects can be ignored for several reasons, for example trying to create something that already exists. Ignores should always be safe, so if something was ignored, it was either not neccesary or it was due to the configuration of the import.
+
+***timingsStats***
+
+`timingStats` represents the time elapsed in different steps of the import. These stats does not provide an accurate overall time for the import, but rather the time spent in the code for different steps.
+
+The `timingStats` are primarily useful for debugging imports that are causing issues, to see which part of the import is having issues.
+```json
+{
+  "timingsStats": {
+    "timers": {
+      "preheat": "0.234086 sec.",
+      "preprocess": "0.000058 sec.",
+      ...
+      "totalImport": "0.236810 sec.",
+      "validation": "0.001533 sec."
+    }
+  }
+}
+```
+
+***bundleReport***
```

Comment:
I think we duplicate a lot that was said previously. We already have a section that described the response (bundleReport)

Reply:
Which section are you referring to here? The idea is that this section should go into details about the importSummary

Reply:
SYNC section, I think. Maybe it makes sense to have responses in both sections, but this one doesn't really add any details.

Reply:
removed the sync example and refer to importsummary section instead